### ISSUE TITLE
uploading a new version of a plugin doesn't update the plugin

### ISSUE THREAD

**Issue by clofresh** [sjrd218/rundeck-verb-project#4]
To reproduce:

* Start from a clean rundeck server, version v3.0.7-78-g0f680e8, rd client version 37f5582
* `rd plugins upload -f 3/db-creds.zip`
* `rd plugins list`
* `rd plugins install --id 9124d6aee8ab`
* Going to /artifact/index shows the plugin is at version 3 and installed.
* Check in the UI that there's a step plugin, RestartApp with `process` and `health_url` parameters with descriptions
* `rd plugins upload -f 4/db-creds.zip`
* `rd plugins list`
* `rd plugins install --id 9124d6aee8ab`
* Going to /artifact/index shows that the version changed to 4
* Check in the UI that there's a step plugin, RestartApp with `process` and `health_url` parameters with descriptions. The descriptions should say "CHANGED" but they are still the same as the original version's

plugin version:

[versions.zip](https://github.com/rundeck/repository/files/2511973/versions.zip)

**Comment by sjrd218** [comment]
@clofresh I downloaded your zip and followed all of your instructions (fantastic issue reporting) but I was not able to reproduce the issue. 

When I pulled up the UI after installing version 4 it showed me the expected `CHANGED` messages in the description. Also, when I looked at the files on the file system the dates corresponded to the installation times in both `libext` and the `libext/cached` for that plugin, so I can see the files were updated to the later versions by the installation process.
